.github/workflows/Build.yml

```diff
@@ -0,0 +1,31 @@
+name: Build JuliaPhysics Website
+
+on:
+  push:
+    branches: [source]
+    tags: '*'
+  pull_request:
+  schedule:
+    - cron: '15 4 * * 3'
```

Comment:
Is it worth it to rebuild this every week? Especially since this will go stale in 2-3 months anyway.